front-end/src/components/Post/GovernanceSideBar/Referenda/ReferendumVoteInfo.tsx

```diff
@@ -36,6 +38,7 @@ const ReferendumVoteInfo = ({ className, referendumId, threshold }: Props) => {
 
 		if (!apiReady) {
 			console.error('api not ready');
+			setLoadingStatus({ isLoading: false, message: 'Api not ready' });
```

Comment:
same here, this doesn't need to be displayed here. "Loading votes" is fine. If we wanted to be more precise, we could have initiated the "Loading api", and then if(api) -> "Loading votes". But since it should be relatively quick, one indication is enough.